pyproject.toml

```
@@ -109,7 +109,7 @@ filterwarnings = [
 
 [tool.ruff]
 line-length = 88
-extend-exclude = ["doc/*", "benchmarks/*", "versioneer.py", "geopandas/_version.py"]
+extend-exclude = ["doc/*", "benchmarks/*", "geopandas/_version.py"]
```

```suggestion
extend-exclude = ["doc/*", "benchmarks/*"]
```

The _version.py no longer exists as well?